python/ray/serve/handle.py

```diff
@@ -208,15 +211,27 @@ def _remote(
     def __getattr__(self, name):
         return self.options(method_name=name)
 
-    def shutdown(self):
+    def shutdown(self, _skip_asyncio_check: bool = False):
```

Comment:
hm is this risky? I doubt anyone's really using it but `shutdown` is still a public api. If anyone's using this on asyncio loops, this is not backwards compatible.

Reply:
The change is backwards compatible because the else condition would only be evaluated when _run_router_in_separate_loop=False, which is a deliberate decision the user needs to make.